src/app/components/MovieCard.tsx

```diff
@@ -1,4 +1,4 @@
-import React from "react";
```

Comment:
Standard React Importstatement wird doch benötigt, oder?

Reply:
jo, ist noch da. In einer anderen Zeile zusammen mit anderen Imports  :)